models/public/mobilefacedet-v1-mxnet/mobilefacedet-v1-mxnet.md

```diff
@@ -0,0 +1,104 @@
+# mobilefacedet-v1-mxnet
+
+## Use Case and High-Level Description
+
+  MobileFace Detection V1 is a Light and Fast Face Detector for Edge Devices (LFFD) model based on Yolo V3 architecture and trained with MXNet\*. For details see the [repository](https://github.com/becauseofAI/MobileFace) and [paper](https://arxiv.org/pdf/1904.10633.pdf).
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Detection     |
+| GFLOPs            | 3.5456       |
+| MParams           | 7.6828        |
+| Source framework  | MXNet\*       |
+
+## Input
+
+### Original model
+
+Image, name - `data`, shape - `[1x256x256x3]`, format -`[BxHxWxC]` where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Expected color order -  `BGR`.
+
+### Converted model
+
+The converted model has the same parameters as the original model.
+
+    WARNING: Note that input layout of converted model is `[BxHxWxC]`.
```

Comment:
I think you need to highlight it using markdown features. This  appeal sound rude. Possibly we can make it more soft if it will start with `Please note` or `Please pay attention` instead of  just Note

Reply:
please review again